.github/CODEOWNERS

```diff
@@ -361,6 +361,7 @@ packages/kbn-eslint-plugin-eslint @elastic/kibana-operations
 packages/kbn-eslint-plugin-imports @elastic/kibana-operations
 packages/kbn-eslint-plugin-telemetry @elastic/actionable-observability
 x-pack/test/encrypted_saved_objects_api_integration/plugins/api_consumer_plugin @elastic/kibana-security
+src/plugins/event_annotation_application @elastic/kibana-visualizations
 packages/kbn-event-annotation-common @elastic/kibana-visualizations
```

Comment:
Sounds great. Happy to change the name. For the record, the reason I created it at all was to avoid circular plugin dependency because we have to consume the embeddable (lens -> annotations -> lens).

Reply:
Yes I remember, let's rename it. It will be easier to understand which code this plugin holds!